src/main/java/de/tum/in/www1/artemis/service/compass/utils/SimilarityFunctions.java

```diff
@@ -0,0 +1,34 @@
+package de.tum.in.www1.artemis.service.compass.utils;
+
+import java.util.Objects;
+
+import de.tum.in.www1.artemis.service.compass.umlmodel.UMLElement;
+
+public class SimilarityFunctions {
```

Comment:
There's a codacy warning here. Would it be better to rename is to `SimilarityUtils` since the file is located in the utilities folder?